src/Hangman/HangmanGame.java

```diff
@@ -0,0 +1,208 @@
+
+    private static int incorrectGuesses = 0;
+
+    private static int correctGuesses = 0;
+    private static HashSet<String> lettersGuessed;
+
+
+
+    public HangmanGame(){
+        board = new String[]{"  ┌──┐",
+                             "     │",
+                             "     │",
+                             "     │",
+                             "   ──┴──"};
+        lettersGuessed = new HashSet<String>();
+    }
+
+    public void printBeginningBoard(){
+        System.out.println("  ┌──┐");
+        System.out.println("  O  │      Start game? (y/n)");
+        System.out.println(" /|\\ │");
+        System.out.println(" / \\ │");
+        System.out.println("   ──┴──");
+    }
+    public void printEndBoard(){
+        System.out.println("  ┌──┐");
+        System.out.println("  O  │      Thank you for playing!");
+        System.out.println(" /|\\ │");
+        System.out.println(" / \\ │");
+        System.out.println("   ──┴──");
+    }
+
+    public void startGame(){
+        printBeginningBoard();
+        Scanner input = new Scanner(System.in);
+        String choice = input.nextLine();
+        if(choice.toLowerCase().equals("n")) printEndBoard();
+        else if (choice.toLowerCase().equals("y")) playGame();
+        else {
+            System.out.println("You must pick either Y or N!");
+            startGame();
+        }
+    }
+
+    public void printBoard(){
+        for (String line : board) {
+            System.out.println(line);
+        }
+    }
+
+    public void printLetters(){
+        for (String toGuess : lettersToGuess) {
+            System.out.print(toGuess);
+        }
+        messageToPlayer();
+    }
+
+    public void playGame(){
+
+        Scanner input = new Scanner(System.in);
+        word = getRandomWord(wordList);
+
+        System.out.println("Press 1 to play the game or Press 2 for testing purposes");
+        String playOrTest = input.next();
+
+        switch (playOrTest){
+            case "1":
+                System.out.println("Enjoy the game!");
+                break;
+            case "2":
+                System.out.println("The word is: " + word);
+                break;
+            default:
+                System.out.println("Input not recognized... oh well the games starting anyway");
+                break;
+        }
+
+        lengthOfWord = word.length();
+        lettersToGuess = new String[lengthOfWord];
+        for (int i = 0; i < lengthOfWord; i++) {
+            lettersToGuess[i] = " - ";
+        }
+        board = new String[]{"  ┌──┐",
+                             "     │",
+                             "     │",
+                             "     │",
+                             "   ──┴──"};
+
+        while(incorrectGuesses < 6 & correctGuesses < lengthOfWord){
+            printBoard();
+            printLetters();
+            String guess = input.next();
+            while(!verifySingleLetterGuess(guess)){
```

Comment:
So a thing i like to do with my methods that return a boolean is to prefix the name of the method with either 'is' or 'does'. That way its easy to see that this is a method that returns a boolean. So for instance instead of 'verifySingleLetterGuess' i would call it something like 'isUsersInputASingleLetter'. Once again not a biggie in this project. But i had to look into the method to see what it was doing. Rather if the name was more like english, i could assume exactly what this method is asking for and automatically knew it returned a boolean just by the name. 

Or maybe this is totally subjective idk i havent talked to other programmers about this specific case of boolean methods